fzf-zsh-plugin.plugin.zsh

```diff
@@ -81,7 +81,7 @@ if [[ -z "$FZF_DEFAULT_COMMAND" ]]; then
   # If fd command is installed, use it instead of find
   _fzf_has 'fd' && _fd_cmd="fd"
   _fzf_has 'fdfind' && _fd_cmd="fdfind"
-  if [[ -n $_fd_cmd ]] then
+  if [[ -n "$_fd_cmd" ]]; then
```

Comment:
Wasn't that a syntax error? (the missing `;`) 🤔

Reply:
I think you're right, thanks for the fix